### PR TITLE
Remove unused callback from GUI exercise

### PR DESCRIPTION
**Pull request by njr0**
Here's a possible update to the GUI/TUI exercise, taking out the callback and explaining (slightly).

See what you think.

### CODE REVIEW

File: src/exercises/day-3/simple-gui.rs [hunk 1]
```diff
@@ -43,14 +43,14 @@ impl Label {
 
 pub struct Button {
     label: Label,
-    callback: Box<dyn FnMut()>,
+    // callback: Box<dyn FnMut()>, // Would also take a callback
```
Comment (mgeisler) [inline src/exercises/day-3/simple-gui.rs:46]: Thanks! Please remove the comment too to make the exercise more focused for the students. It's already helpful that you mention the possibility for a callback in the text above.

**Review by mgeisler — CHANGES_REQUESTED** [review]
This is nice, we should just take away the comments from the code.

**Comment by njr0** [comment]
That's done. At least, I've updates the code in the branch and re-pushed. TBH, I don't use pull requests all that much, so I'm not certain whether I have to do anything else, but I can see the update on the main repo, so I presume you can too?

File: src/exercises/day-3/safe-ffi-wrapper.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 Rust has great support for calling functions through a _foreign function
 interface_ (FFI). We will use this to build a safe wrapper for the `libc`
-functions you would use from C to read the filenames of a directory.
+functions you would use from C to read the names of files in a directory.
```
Comment (mgeisler) [inline src/exercises/day-3/safe-ffi-wrapper.md:5]: This change is unrelated and has already been reviewed in #1286. You should create a new branch for every change and avoid making changes "on top" of each other:

```suggestion
functions you would use from C to read the filenames of a directory.
```

**Comment by mgeisler** [comment]
> That's done. At least, I've updates the code in the branch and re-pushed. TBH, I don't use pull requests all that much, so I'm not certain whether I have to do anything else, but I can see the update on the main repo, so I presume you can too?

The push is good, I also see the new changes here.

The only missing thing is the CLA bot (both here and in your other pull request). When the CLA is signed, then we can merge the PRs.

**Comment by njr0** [comment]
I've signed a CLA.

**Comment by mgeisler** [comment]
> I've signed a CLA.

Great, I told the bot to run the check again and it all looks good now. Thanks a lot for the help!

File: src/exercises/day-3/simple-gui.md [hunk 3]
```diff
@@ -1,13 +1,14 @@
...
-trait objects.
+trait objects. We'll only implement the drawing of it (as text) for simplicity.
 
 We will have a number of widgets in our library:
 
 * `Window`: has a `title` and contains other widgets.
-* `Button`: has a `label` and a callback function which is invoked when the
-  button is pressed.
+* `Button`: has a `label`. In reality, it would also take a callback
+  to allow the called to do something when the button is clicked
```
Comment (mgeisler) [inline src/exercises/day-3/simple-gui.md:10]:
```suggestion
  function to allow the program to do something when the button is clicked
```

**Comment by mgeisler** [comment]
> Sorry, typo. I mean caller, not called!

Ah, I did a best-effort on-the-fly update :smile: Feel free to submit a patch for this (I recommend using the little pencil icon in the top-right of the [Drawing A Simple GUI](https://google.github.io/comprehensive-rust/exercises/day-3/simple-gui.html#drawing-a-simple-gui) page).
